pom.xml

```diff
@@ -9,7 +9,6 @@
 
   <name>Kubernetes CLI Plugin</name>
   <url>https://github.com/jenkinsci/kubernetes-cli-plugin</url>
-  <groupId>org.jenkins-ci.plugins</groupId>
```

Comment:
Why is the `groupId` not needed anymore ?

Reply:
there was a warning because the group took it from the parent